qan-api2/models/metrics.go

```diff
@@ -1026,3 +1026,102 @@ func (m *Metrics) ExplainFingerprintByQueryID(ctx context.Context, serviceID, qu
 
 	return res, errors.New("query_id doesnt exists")
 }
+
+const selectedQueryMetadataTmpl = `
+SELECT 	any(service_name), any(database), any(schema), any(username), 
+		any(replication_set), any(cluster), any(service_type), any(service_id), 
+		any(environment), any(node_id), any(node_name), any(node_type) 
+FROM 	metrics
+WHERE 	period_start >= :period_start_from 
+	AND period_start <= :period_start_to {{ if not .Totals }} 
```

Comment:
usually end is excluding, @JiriCtvrtka what do you think?

Reply:
@ritbl You are asking on "last" minute from period to? If yes  this is OK, we using this period where clause in same way on more places.